decentraland/unity-explorer
# fix: initial notification read after every successful login

## What does this PR change?


Fix #2159 

Fetch user notifications every time a new login is made in order to jump in game with the correct notifications loaded 

## How to test the changes?



1. Launch the explorer
2. Login with an account and enter the Explorer
3. Verify which notifications you have
4. Logout
5. Launch the explorer again
6. Login with a different account
7. Enter in game and verify that different notifications are loaded

You can verify the behavior with unread notifications as well.
Refer to [this](https://github.com/decentraland/unity-explorer/pull/2148) PR in order to push new notifications to an account.

1. Push some notifications to an account A
2. Push some notifications to an account B (a different amount than A)
3. Login with account A, enter the Explorer and verify that the number of unread notifications matches with the number of pushed ones
4. Logout
5. Login with account B, enter the Explorer and verify that the number of unread notifications matches with the number of pushed ones (different than A)

For a double check, at the login screen, you can also logout/login multiple times with different accounts and then enter the Explorer verifying that the notifications shown are coherent with the account you choose to enter in game.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
